x-pack/legacy/plugins/ml/public/data_frame/common/navigation.ts

```diff
@@ -10,11 +10,12 @@ export function moveToDataFrameWizard() {
   window.location.href = '#/data_frames/new_transform';
 }
 
-export function moveToDataFrameTransformList() {
-  window.location.href = '#/data_frames';
-}
-
-export function moveToDiscover(indexPatternId: string, baseUrl: string) {
+/**
+ * Gets a href for navigating to Discover page.
+ * @param indexPatternId Index pattern id.
+ * @param baseUrl Base url.
+ */
+export function getDiscoverHref(indexPatternId: string, baseUrl: string): string {
```

Comment:
naming nit pick, i think this should be `getDiscoverUrl`.
to me, `getDiscoverHref` isn't clear what it will return.